Formula/synergy-core.rb

```diff
@@ -0,0 +1,33 @@
+class SynergyCore < Formula
+  desc "Open source core of Synergy, the keyboard and mouse sharing tool"
+  homepage "https://symless.com/synergy"
+  url "https://github.com/symless/synergy-core.git",
+      :revision => "62ab8ffc4fdbc84b51552d600a8e11f7a7b78d84"
+  version "1.9.0-rc4"
+  head "https://github.com/symless/synergy-core.git"
+
+  depends_on "cmake" => :build
+  depends_on "qt"
+  depends_on "openssh"
+
+  def install
+    mkdir "build" do
+      system "cmake",
```

Comment:
please use std_cmake_args. none of the custom args here should be needed as we already set the relevant environment variables